### PR TITLE
DE37821 flexbox and animations

### PR DESCRIPTION
**Pull request by ChrisKraljevicD2L**
Requires [this sequences PR](https://github.com/BrightspaceHypermediaComponents/sequences/pull/181)

### CODE REVIEW

File: d2l-sequence-viewer.js [hunk 1]
```diff
@@ -504,43 +547,27 @@ class D2LSequenceViewer extends mixinBehaviors([
...
-			this.$.viewframe.style.marginLeft = marginLeft;
-			this.$.viewframe.style.marginRight = String(offsetWidth) + 'px';
-		} else {
-			this.$$('#sidebar').style.width = '310px';
-			this.$.viewframe.style.marginLeft = 'auto';
-			this.$.viewframe.style.marginRight = String(offsetWidth) + 'px';
-		}
-		this.$.sidebar.classList.remove('offscreen');
+		const sidebarContainer = this.shadowRoot.getElementById('sidebar-container');
+		sidebarContainer.classList.remove('offscreen');
```
Comment (ianlaidlaw) [inline d2l-sequence-viewer.js]: 👌 much better

File: d2l-sequence-viewer.js [hunk 2]
```diff
@@ -504,43 +547,27 @@ class D2LSequenceViewer extends mixinBehaviors([
 	}
 
 	_sideBarOpen() {
-		const maxWidth = 1170;
-		const sidebarWidth = Math.round((this.offsetWidth <= maxWidth ? this.offsetWidth : maxWidth) / 3);
-		const offsetWidth = this.$$('#sidebar-occlude').offsetWidth;
-		const marginLeft = `${sidebarWidth + offsetWidth}px`;
-		if (this.mEntity && this.mEntity.properties
```
Comment (ianlaidlaw) [inline d2l-sequence-viewer.js]: this if statement will not be missed

**Comment by ianlaidlaw** [comment]
not a huge problem, but screenshots or gifs in the pr will be ideal 👍

**Comment by ChrisKraljevicD2L** [comment]
navbar_movement

navbar_movement_2

**Comment by ianlaidlaw** [comment]
everything looks good - only thing I would want to double check is, how does this look on iOS 13 with quizzes? the changes here mostly seem to be about "width" so it probably shouldn't affect the scrolling to the bottom issue i think!

**Comment by ChrisKraljevicD2L** [comment]
Good idea. I'll give it a test.

**Pull request merged**